### ISSUE TITLE
Not possible to add Folders / Items

### ISSUE THREAD

**Issue by apollo79** [BigE/desk-changer#94]
I wanted to add a folder with images as a source to the profile, so I removed the default Location URI, and tried to add it, but nothing happens if I click any of the buttons (`Add Item(s)`, `Add Folder(s)`, `Add New Profile`). `extensions.gnome.org` shows me, that there was an error.

Operating System: Zorin v16.2
![desk_changer](https://user-images.githubusercontent.com/90517340/212465701-1399f56e-7cf9-448f-a4c4-d7c187ab4d8b.png)
![desk_changer_error](https://user-images.githubusercontent.com/90517340/212465703-2631fc34-9f0f-4da8-8405-e277ce668db1.png)

**Comment by BigE** [comment]
Please check your metadata.json and see what version you have. This should have been fixed in the most recent version ([35](https://github.com/BigE/desk-changer/releases/tag/version-35) from github, 25 from [extensions.gnome.org](https://extensions.gnome.org/extension/1131/desk-changer/)) by [this commit](https://github.com/BigE/desk-changer/commit/f2395f642921c8b68f1e4220749b5eaf83ab856b)

If you've got the latest version and still have a problem, please let me know and I'll look into it when I can. Thanks for the report!

**Comment by apollo79** [comment]
Thanks for the fast response! 
I reinstalled and checked the version, it is 25, but sadly, still nothing happens when I click the buttons.

**Comment by BigE** [comment]
What version of gnome-shell are you using?

**Comment by apollo79** [comment]
> What version of gnome-shell are you using?

I am currently on `GNOME Shell 3.38.4`

**Comment by BigE** [comment]
Any chance you've upgraded gnome shell? I've fixed this for [version 45](https://github.com/BigE/desk-changer/releases/tag/version-36) and could potentially backport the fix to older versions. Let me know what version you're on, if not 45, and I'll see what I can do. Thanks!

**Comment by apollo79** [comment]
No, I am using [Zorin OS](https://zorin.com/os), and it ships with gnome shell 43 unfortunately :(
